makefile

```diff
@@ -5,7 +5,7 @@ EXEC = emerald
 CCFLAGS = -std=c++11 -DGL_GLEXT_PROTOTYPES -DGLEW_STATIC -DGLM_FORCE_RADIANS -D_DEFAULT_SCENE_FILE_="\"scene_file.txt\""
 INC =
 LDFLAGS = -lSDL2 -lSDL2_image -lassimp
-FRAGMENT_SHADER_PRECOMPILE= "_MAX_LIGHTS_=100"
+FRAGMENT_SHADER_PRECOMPILE= "_MAX_LIGHTS_=100" "_NEAR_=0.1" "_FAR_=100"
```

Comment:
This should be set in makefile so that both GPU shaders and CPU code use the same values.